foundation.cabal

```diff
@@ -111,6 +111,7 @@ library
                      Foundation.UUID
                      Foundation.System.Entropy
                      Foundation.System.Bindings
+                     Foundation.System.Bindings.POSIX.Stropts
```

Comment:
I would rather see this still in basement. I appreciate there's a bits of a complication with the dependencies, but I'ld the basic terminal stuff as low as possible to get away with building a good abstraction directly in basement (which need to do some terminal stuff, e.g. `error`)